packages/now-cli/src/commands/deploy/latest.js

```diff
@@ -732,7 +732,7 @@ function handleCreateDeployError(output, error) {
 
       output.error(
         `The property ${code(prop)} is not allowed in ${highlight(
-          'now.json'
+          'vercel.json'
         )} when using Now 2.0 – please remove it.`
```

Comment:
```suggestion
        )} – please remove it.`
```

Reply:
At first I was thinking we should keep this since 1.0 isn't going to be removed until June 1st but then I realized that we don't want to print "Now" anywhere. Which makes me think there are still tons of places that need to update like all the help menus: `now env`, `now ls`, etc 🤦‍♂️